.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: c
 
-sudo: false
+sudo: required
```

Comment:
is this still required?

Reply:
It should not be needed, if the installation from repositories worked (the last build was success including the installation of cmocka, so I will remove this line).